src/main/java/net/sf/jabref/util/MassSetFieldAction.java

```diff
@@ -150,7 +164,7 @@ public void stateChanged(ChangeEvent e) {
             public void actionPerformed(ActionEvent e) {
                 // Check if the user tries to rename multiple fields:
                 if (rename.isSelected()) {
-                    String[] fields = getFieldNames(field.getText());
+                    String[] fields = getFieldNames((String) field.getSelectedItem());
```

Comment:
hm, strange that a generically typed combo box requires a cast to String. Is this really necessary?